envi/archs/amd64/opcode64.py

```diff
@@ -737,7 +737,7 @@
 tbl32_660F[0xd4] = (0, INS_OTHER, ADDRMETH_V | OPTYPE_x | OP_W, ADDRMETH_VEXSKIP | ADDRMETH_H | OPTYPE_x | OP_R, ADDRMETH_W | OPTYPE_x | OP_R, ARG_NONE, cpu_PENTMMX, "paddq", 0, 0, 0)
 tbl32_660F[0xd5] = (0, INS_OTHER, ADDRMETH_V | OPTYPE_x | OP_W, ADDRMETH_VEXSKIP | ADDRMETH_H | OPTYPE_x | OP_R, ADDRMETH_W | OPTYPE_x | OP_R, ARG_NONE, cpu_PENTMMX, "pmullw", 0, 0, 0)
 tbl32_660F[0xd6] = (0, INS_OTHER, ADDRMETH_W | OPTYPE_q | OP_W, ADDRMETH_V | OPTYPE_q | OP_R, ARG_NONE, ARG_NONE, cpu_PENTMMX, "movq", 0, 0, 0)
-tbl32_660F[0xd7] = (0, INS_OTHER, ADDRMETH_G | OPTYPE_q | OP_W, ADDRMETH_U | OPTYPE_x | OP_R, ARG_NONE, ARG_NONE, cpu_PENTIUM2, "pmovmskb", 0, 0, 0)
+tbl32_660F[0xd7] = (0, INS_OTHER, ADDRMETH_G | OPTYPE_d | OP_W, ADDRMETH_U | OPTYPE_x | OP_R, ARG_NONE, ARG_NONE, cpu_PENTIUM2, "pmovmskb", 0, 0, 0)
```

Comment:
Uhhh, perhaps the merging in of master didn't go all the way through due to conflicts?

Because master should be `OPTYPE_q`, but your branch is showing `OPTYPE_d`.

Reply:
Oh. No. Did you merge in my bugfix branch that's still out for review?

Reply:
i don't *think* so!?  i'm pretty sure i've only been merging in master.

Reply:
Oh, no, I think you did merge in #333. I see the noret analysis pass down below. And if you look at the git log for this branch, you'll see my commits (like 02c8998) interspersed with your own.

I'm not sure if it'll just fix itself when #333 gets merged, or it'll resign you to git hell. We might wanna walk through this one together.